shared-bindings/busio/I2C.c

```diff
@@ -67,10 +67,11 @@
 STATIC mp_obj_t busio_i2c_make_new(const mp_obj_type_t *type, size_t n_args, size_t n_kw, const mp_obj_t *all_args) {
     busio_i2c_obj_t *self = m_new_obj(busio_i2c_obj_t);
     self->base.type = &busio_i2c_type;
-    enum { ARG_scl, ARG_sda, ARG_frequency, ARG_timeout };
+    enum { ARG_scl, ARG_sda, ARG_setpullup, ARG_frequency, ARG_timeout };
```

Comment:
```suggestion
    enum { ARG_scl, ARG_sda, ARG_pullup, ARG_frequency, ARG_timeout };
```
Just `pullup` should be fine.

Reply:
I would suggest `internal_pullup`, to make it clear we're not say that there is _some_ kind of pullup.
```suggestion
    enum { ARG_scl, ARG_sda, ARG_internal_pullup, ARG_frequency, ARG_timeout };
```